ecs-init/docker/docker.go

```diff
@@ -79,11 +79,6 @@ const (
 	// For more information on setns, please read this manpage:
 	// http://man7.org/linux/man-pages/man2/setns.2.html
 	CapSysAdmin = "SYS_ADMIN"
-	// CapChown to start agent with CAP_CHOWN capability
```

Comment:
in addition this, should we move the list of all capabilities into its own little block and add a comment that a prefix `CAP_` is appended to all the below capabilities by default?

Reply:
I changed this to a revert commit. Let's add it when we introduce it again.

Reply:
this will prevent us from making the same mistake again, not just for `CHOWN` but all capabilities in general.

Reply:
Yep I got that. I was just trying to avoid doing more changes and get the fix out asap. I will add that since it's just a comment.